From ed0bfd14a05a56c05b2d895db88645c28c681346 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Wed, 24 May 2017 22:59:47 -0700
Subject: [PATCH] Add <functional> header for std::functional

gcc 7.1 does not include this header indirectly
as older versions

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Source/WTF/wtf/StdLibExtras.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/WTF/wtf/StdLibExtras.h b/Source/WTF/wtf/StdLibExtras.h
index 5f388eb4f0f..25b0f86eafa 100644
--- a/Source/WTF/wtf/StdLibExtras.h
+++ b/Source/WTF/wtf/StdLibExtras.h
@@ -30,6 +30,7 @@
 #include <chrono>
 #include <cstring>
 #include <memory>
+#include <functional>
 #include <wtf/Assertions.h>
 #include <wtf/CheckedArithmetic.h>
 #include <wtf/Compiler.h>
-- 
2.13.0

